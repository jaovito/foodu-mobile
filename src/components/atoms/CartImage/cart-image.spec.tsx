import {render} from '@test-utils';
import {CartImage} from './cart-image';
import defaultImage from '../../../assets/pizza.png';

describe('<CartImage /> tests', () => {
  it('should be defined', () => {
    const image = render(<CartImage source={defaultImage} />);

    expect(image).toBeDefined();
  });

  it('should be able to match with snapshot', () => {
    const image = render(<CartImage source={defaultImage} />).toJSON();

    expect(image).toMatchSnapshot();
  });
});
