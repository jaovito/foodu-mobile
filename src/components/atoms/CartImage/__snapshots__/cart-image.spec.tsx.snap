// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CartImage /> tests should be able to match with snapshot 1`] = `
<Image
  source={
    {
      "testUri": "../../../src/assets/pizza.png",
    }
  }
  style={
    [
      {},
    ]
  }
/>
`;
